sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -291,6 +298,12 @@ def _get_headers(headers, request_type, replica_count, uses_feature_flags, featu
     if replica_count > 0:
         correlation_context += ",ReplicaCount=" + str(replica_count)
 
+    if uses_load_balancing:
+        correlation_context += ",UsesLoadBalancing"
```

Comment:
We missed adding this tag in .NET provider so we get to decide the name here and .NET will follow :)
How about "UsesLB" to keep it short?

cc @zhiyuanliang-ms

Reply:
Looks like it is already just `LB`? https://github.com/Azure/AppConfiguration-DotnetProvider/blob/90835cf5efbfb26d6de6068e4c31c3050947b78a/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs#L33

Reply:
You're right. I forgot that we added a new "Features" tag in .NET provider. So that means we need to do the same here. If load balancing is enabled, we need to write `Features=LB` to correlation context header.

https://github.com/Azure/AppConfiguration-DotnetProvider/blob/90835cf5efbfb26d6de6068e4c31c3050947b78a/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs#L184C13-L187C14

Reply:
Updated